docs/lang/articles/basic/sparse.md

```diff
@@ -363,6 +363,48 @@ print(ti.rescale_index(block2, block1, [3, 1]))       # output: [1, 0]
 
 Regarding line 1, you can also compute the `block1` index given `pixel` index `[7, 3]` as `[7//2//2, 3//2//2]`. However, doing so couples computation code with the internal configuration of data structures (in this case, the size of `block1` containers). By using `ti.rescale_index()`, you can avoid hard-coding internal information of data structures.
 
+
+## Sparse grid example
+We now show a simple example of how to use the sparse data structures to implement a sparse grid.
+
+```python
+import taichi as ti
+ti.init()
+grid_size = (10,10)
+@ti.data_oriented
+class SparseGrid():
+    def __init__(self, grid_size):
+        self.mass = ti.field(float)
+        self.pos = ti.Vector.field(3, float)
+        self.snode = ti.root.bitmasked(ti.ij, grid_size)
+        self.snode.place(self.mass)
+        self.snode.place(self.pos)
+    @ti.kernel
+    def usage(self):
+        cnt = 0
+        for I in ti.grouped(self.snode):
+            if ti.is_active(self.snode, I):
+                cnt+=1
+        usage =  cnt/(grid_size[0]*grid_size[1])
+        print("Grid usage: ", usage)
+sp = SparseGrid(grid_size=grid_size)
```

Comment:
@chunleili is it easier to just use ti.sparse.grid that you just added?

Reply:
Yes, but it is not released yet. So add the doc of this new API now may be confusing

Reply:
@chunleili the doc you're modifying will be shown on a `develop` branch on the doc site (which tracks master branch so it's totally fine to use it here ;)